packages/dds/pact-map/src/interfaces.ts

```diff
@@ -31,6 +52,12 @@ export interface IPactMap<T = unknown> extends ISharedObject<IPactMapEvents> {
 	 */
 	get(key: string): T | undefined;
 
+	/**
+	 * Gets the accepted value for the given key.
+	 * @param key - The key to retrieve from
+	 */
+	getDetails(key: string): IAcceptedPactDetails<T> | undefined;
```

Comment:
Maybe `getWithDetails()`?  I interpret `getDetails()` as that it will only get the details and not the value.